client/src/components/Table.js

```diff
@@ -0,0 +1,108 @@
+import React, {Component} from "react";
+import "../css/Table.css";
+import "../css/themePalette.css";
+import { Table } from "react-bootstrap";
+
+
+
+// The Table
+class LogTable extends Component {
+  constructor() {
+    super();
+    // Made dummy data as state becuase it might be easier to change later
+    this.state = {
+      questionnaire: [
+        "Date",
+        "How Do you Feel?",
+        "How Are My Symptoms?",
+        "Difference From Yesterdsy",
+        "Any Noted Symptoms",
+        "Comments"
+      ],
+      log: [
```

Comment:
We have already the `Observation` model. You should derive the display model from these.